modules/@apostrophecms/doc-type/index.js

```diff
@@ -422,22 +416,9 @@ module.exports = {
       },
       getRelatedDocsIds(req, doc) {
         const relatedDocsIds = [];
-        const can = (field) => {
```

Comment:
I assume this is you backing out your own new logic.

Reply:
Yep